docs/pages/access-controls/device-trust/device-management.mdx

```diff
@@ -0,0 +1,381 @@
+are automatically updated in Teleport device inventory, avoiding manual device operation.
+</Admonition>
+
+Before you can enroll the device, you need to register it. To register
+a device, you first need to determine its serial number.
+
+<Tabs>
+  <TabItem label="macOS">
+    The serial number of a macOS device can be determined in two ways.
+
+    Through the UI, visible under Apple -> "About This Mac" -> "Serial number".
+
+    Through the terminal:
+    ```code
+    $ ioreg -c IOPlatformExpertDevice -d 2 | grep -i IOPlatformSerialNumber | awk -F'"' '{print $4}'
+    (=devicetrust.asset_tag=)
+    ```
+
+    Replace <Var name="(=devicetrust.asset_tag=)" description="The serial number to be registered"/>
+    with the serial number obtained from the device you wish to enroll and run `tctl devices add` command:
+
+    ```code
+    $ tctl devices add --os=macos --asset-tag="<Var name="(=devicetrust.asset_tag=)"/>"
+    Device <Var name="(=devicetrust.asset_tag=)"/>/macos added to the inventory
+    ```
+
+    Use `tctl` to check that the device has been registered:
+
+    ```code
+    $ tctl devices ls
+    Asset Tag    OS    Enroll Status Device ID
+    ------------ ----- ------------- ------------------------------------
+    (=devicetrust.asset_tag=) macOS not enrolled  (=devicetrust.device_id=)
+    ```
+  </TabItem>
+
+  <TabItem label="Windows">
+    Windows devices can have multiple serial numbers depending on the configuration made by the manufacturer. Teleport will pick the first available value from the following:
+    - System asset tag
+    - System serial number
+    - Baseboard serial number
+
+    To find the value chosen by Teleport, run the following command:
+
+    ```code
+    $ tsh device collect
+    DeviceCollectedData {
+      ...
+      "serial_number": "(=devicetrust.asset_tag=)",
+      ...
+    }
+    ```
+
+    Replace <Var name="(=devicetrust.asset_tag=)" description="The serial number to be registered"/>
+    with the serial number obtained from the device you wish to enroll and run `tctl devices add` command:
+
+    ```code
+    $ tctl devices add --os=windows --asset-tag="<Var name="(=devicetrust.asset_tag=)"/>"
+    Device <Var name="(=devicetrust.asset_tag=)"/>/windows added to the inventory
+    ```
+
+    Use `tctl` to check that the device has been registered:
+
+    ```code
+    $ tctl devices ls
+    Asset Tag    OS    Enroll Status Device ID
+    ------------ ----- ------------- ------------------------------------
+    (=devicetrust.asset_tag=) windows not enrolled  (=devicetrust.device_id=)
+    ```
+  </TabItem>
+</Tabs>
+
+<Admonition type="warning" title="Device Role">
+For clusters created using Teleport v12 or newer, the builtin role editor has the necessary permissions to manage devices.
+If you Teleport cluster is older than v12, you will need to create and grant `device` role to manage device inventory.[link to below]
```

Comment:
Don't forget to update this `[link to below]`

Reply:
Thanks for spotting that. I swear I had updated that once but it must have resurfaced during copying content from docs preview to this PR branch 😭